parm/config/config.aeroanl

```diff
@@ -5,13 +5,19 @@
 
 echo "BEGIN: config.aeroanl"
 
-export OBS_YAML_DIR=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/config/
-export OBS_LIST=$HOMEgfs/sorc/gdas.cd/parm/aero/obs/lists/aero_prototype.yaml
-export AEROVARYAML=$HOMEgfs/sorc/gdas.cd/parm/aero/variational/3dvar_dripcg.yaml
-export BERROR_YAML=$HOMEgfs/sorc/gdas.cd/parm/aero/berror/static_bump.yaml
-export FV3JEDI_FIX=$HOMEgfs/fix/gdas
+export OBS_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/config/
+export OBS_LIST=${HOMEgfs}/sorc/gdas.cd/parm/aero/obs/lists/gdas_aero_prototype.yaml
+export AEROVARYAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/variational/3dvar_gfs_aero.yaml
+export BERROR_YAML_DIR=${HOMEgfs}/sorc/gdas.cd/parm/aero/berror/
+export BERROR_YAML=${HOMEgfs}/sorc/gdas.cd/parm/aero/berror/staticb_bump_aero.yaml
+export FV3JEDI_FIX=${HOMEgfs}/fix/gdas
+export BERROR_TYPE='identity'
+export BERROR_DATA_DIR=/work/noaa/da/Andrew.Tangborn/staticB/c96/base
```

Comment:
Do we want to move these berror files to a directory in the standard `gdas_fix_root` or is it too soon for this?

Reply:
Eventually, but @andytangborn plans to do a lot of tuning work on **B** in the coming weeks/months for aerosols. So I think my plan (if it is okay with you) is to just support the identity **B** for aerosols in develop for now (i.e. make that the default option). We can put a default dummy path here that would mimic what the real path would look like.

Reply:
Works for me.